kuma/static/js/contribution-handler.js

```diff
@@ -6,46 +6,183 @@
         return;
     }
 
-    // TODO: handle this better
-    var isMobile = $('main').width() < 800;
-
-    if ($().tooltip) {
-        var tooltipButton = $('#email-tooltip'),
-            tooltipCloseButton = $('.tooltip-close');
-        tooltipButton.tooltip({
-            items: '#email-tooltip',
-            content: '<button class="tooltip-close"></button> <span>' + tooltipButton.prev().attr('title') + '</span>',
-            position: {
-                my: isMobile ? 'center bottom' : 'right right',
-                at: isMobile ? 'right top' : 'left left'
+    /**
+     * Expiration date for how long the contributions modal should be hidden for
+     * after it's been disabled by clicking the close button.
+     * @const
+     */
+    var CONTROBUTIONS_DISABLED_EXPIRATION =  5 * 24 * 60 * 60 * 1000; // 5 days.
```

Comment:
I'll change "CONTROBUTIONS_DISABLED_EXPIRATION" to "CONTRIBUTIONS_DISABLED_EXPIRATION".